README.md

```diff
@@ -59,57 +59,119 @@ Three high level components are required to execute a query:
 
 The complete flow is as follows:
 
-```
-        calcite = pyhdk.sql.Calcite(storage)
-        executor = pyhdk.Executor(data_mgr)
-        ra = calcite.process("SELECT * FROM t;")
-        rel_alg_executor = pyhdk.sql.RelAlgExecutor(
-            executor, storage, data_mgr, ra
-        )
-        res = rel_alg_executor.execute()
+```python
+calcite = pyhdk.sql.Calcite(storage)
+executor = pyhdk.Executor(data_mgr)
+ra = calcite.process("SELECT * FROM t;")
+rel_alg_executor = pyhdk.sql.RelAlgExecutor(
+    executor, storage, data_mgr, ra
+)
+res = rel_alg_executor.execute()
 ```
 
 Calcite reads the schema information from storage, and the Executor stores a reference to Data Manager for buffer/storage access during a query. 
 
 The return from RelAlgExecutor is a ResultSet object which can be converted to Arrow and to pandas:
-```
-     df = res.to_arrow().to_pandas()
+```python
+df = res.to_arrow().to_pandas()
 ```
 
 ## Examples
 
 Standalone examples are available in the `examples` directory. Most examples run via Jupyter notebooks. 
 
+
 ## Build
 
-HDK includes components from external libraries as submodules. To clone the project, either use `git clone --recurse-submodules` to initially clone the repo, or clone without using flags and then run:
+### Dependencies 
 
-```
-git submodule init
-git submodule update
+Miniconda installation required. (Anaconda may produce build issues.) Use one of this [miniconda installers](https://docs.conda.io/en/latest/miniconda.html).
+
+Conda environments are used for HDK development. Use the yaml file in `omniscidb/scripts/`:
+
+```bash
+conda env create -f omniscidb/scripts/mapd-deps-conda-dev-env.yml
+conda activate omnisci-dev
 ```
 
-When pulling changes from upstream, be sure to update submodule references using `git submodule update`.
+### Compilation
 
 If using a Conda enviornment, run the following to build and install HDK:
 
-```
-mkdir build
-cd build
-cmake -DENABLE_CONDA=on ..
-make -j
+```bash
+mkdir build && cd build
+cmake ..
+make -j 
 make install
 ```
 
-### Dependencies 
+By default CUDA support disabled.
+
+To verify check `python -c 'import pyhdk'` executed without an error.
+
+#### Compilation with CUDA support
 
-Conda environments are used for HDK development. Use the yaml file in `scripts/conda`:
+##### Dependencies
 
+Install extra dependencies into existing environment or into a new one.
+
+```bash
+conda install -c conda-forge cudatoolkit-dev arrow-cpp-proc=3.0.0=cuda arrow-cpp=8.0=*cuda
 ```
-     conda env create -f scripts/conda/hdk-conda-dev-env.yml
-     conda activate hdk-dev
+
+##### Compilation
+
+```bash
+mkdir build && cd build
+cmake -DENABLE_CUDA=on  ..
+make -j 
+make install
 ```
 
+### Issues
+
+If you meet issues during build refer to [`.github/workflows/build.yml`](https://github.com/intel-ai/hdk/blob/b0bc9ca3f599f648bcc1a8e1eb49c0f02e1f7aca/.github/workflows/build.yml). This file describes compilation steps used for CI build.
```

Comment:
I think a relative link to the file would be more stable here so that you don't need to update it.

Reply:
Thanks, fixed.